### ISSUE TITLE
Stage container can't get focus by touch events

### ISSUE THREAD

**Issue by zarv1k** [konvajs/konva#1172]
Hi!

First of all a wanna say thank you for the great library!

While implementing the focus state for my drawing editor, I noticed that the stage container can't get focus by touches in mobile Safari/Chrome.

I've created an example for reproducing the issue on codesandbox - https://codesandbox.io/s/intelligent-hamilton-grpp7. 
It works well in desktop browsers with clicks, but doesn't in mobile Safari/Chrome on iPhone/iPad and in mobile chrome on Android 11 device. Such a bug makes a bit pain to successfully implement the focus of the editor.

Just `.focusable` div (with tabIndex={1} on the right side) could get focus by touches. 

Let's open it [example](https://grpp7.csb.app) on some touch device to check this out and try touching on blocks to give them focus.

Thank you

**Comment by lavrton** [comment]
You may need to focus it manually. Konva prevents default actions on most of the events. It is required for drag&drop support. So I think it is not focusing automatically because of that.

```js
<Stage
  ref={ref}
  className="container"
  width={200}
  height={200}
  tabIndex={1}
  onTouchStart={(e) => {
    e.target.getStage().container().focus();
  }}
>
```

https://codesandbox.io/s/stoic-ives-xyset?file=/src/index.js

**Comment by zarv1k** [comment]
@lavrton thanks for the quick response.

> It is required for drag&drop support. So I think it is not focusing automatically because of that. 

I think there should be a way to disable that because e.g. in "free drawing mode" of some editor implementation I don't need d&d feature turned on at some moment e.g. while free drawing.
There is `Node.preventDefault` attr, but it doesn't work in this case somehow...

**Comment by lavrton** [comment]
> I think there should be a way to disable that because I don't need d&d feature turned on at some moment e.g. while free drawing

You may still want to prevent default action, is it may trigger native scrolling. Most likely you don't need it.

**Comment by zarv1k** [comment]
> You may still want to prevent default action, is it may trigger native scrolling. Most likely you don't need it.

Yes, but in this case I could prevent the default action in my client's code in event handler, don't I? 
If I used `preventDefault=false` I would like to expect that I must handle all the stuff in my own handler as I want to. But unfortunately from the developer side I can't do it.

**Comment by lavrton** [comment]
That makes sense. I need to think how to update Konva in a better way to handle that.

**Comment by zarv1k** [comment]
> I need to think how to update Konva in a better way to handle that.
Great, thank you, so could you then reopen the issue to track changes on this problem?

**Comment by zarv1k** [comment]
> That makes sense. I need to think how to update Konva in a better way to handle that.

@lavrton Hi!

This issue has been closed w/o solution.
So I don't understand why this issue is closed, coz the problem still exist.
I was notified that the issue #1180 created for rethinking and fixing `preventDefault` behavior has been closed too w/o any solution. 
Would you mind please return back to this issue some time?

**Comment by zarv1k** [comment]
I seems it has been fixed in #1372

Container now receives focus on touch devices using konva 8.3.14 - [sandbox](https://codesandbox.io/s/hardcore-ioana-4jf77t)
